server/server.js

```diff
@@ -1,21 +1,65 @@
 const express = require('express');
+// destructured import of mongoose.Mongoose object
+const {Mongoose} = require('mongoose');
 const app = express();
 const path = require('path');
 
+// gets mediaController middleware from mediaController.js
+const mediaController = require('./../controllers/mediaController');
 
 // use express.json instead of body-parser
```

Comment:
I think you will also need the cors() middleware https://expressjs.com/en/resources/middleware/cors.html

The usage would just be app.use(cors()). To understand why this is needed, you can read the intro to this MDN document: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

Basically, this global middleware is required to indicate that a cross-domain request (from localhost:8080 to localhost:3000) is permissible or something like that.